<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsartAssistant</class>
 <widget class="QWidget" name="UsartAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UsartAssistant</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,2,5,0,10" columnstretch="1,1">
     <item row="4" column="0">
      <widget class="QTextEdit" name="SerialWrite_textEdit">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="PhotoShow_label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Photo...</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="SendData">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>SendData</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="Tools_groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="title">
        <string>Tools</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="OpenSerialPort_label">
              <property name="text">
               <string>OpenSerialPort</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="Com_comboBox">
              <property name="cursor">
               <cursorShape>ArrowCursor</cursorShape>
              </property>
              <property name="mouseTracking">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="OpenCamera_pushButton">
            <property name="text">
             <string>OpenCamera</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="ScanSerialPort_pushButton">
            <property name="text">
             <string>ScanSerialPort</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="CloseCamera_pushButton">
            <property name="text">
             <string>CloseCamera</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="OpenImage_pushButton">
              <property name="text">
               <string>OpenImage</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="SerialSendTest_pushButton">
              <property name="text">
               <string>SendTest</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="TakePhotos_pushButton">
            <property name="text">
             <string>TakePhotos</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" rowspan="2">
      <widget class="QTextEdit" name="SerialRead_textEdit">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ReceiveData">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>ReceivedData</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="2">
      <widget class="QGroupBox" name="Functions_groupBox">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="title">
        <string>Functions</string>
       </property>
       <widget class="QPushButton" name="FindYellowBlock_pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>61</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>黄木块</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
